Library/Homebrew/debrew.rb

```diff
@@ -111,7 +111,7 @@ def self.debug(e)
 
     begin
       puts "#{e.backtrace.first}"
-      puts "#{Tty.red}#{e.class.name}#{Tty.reset}: #{e}"
+      puts Formatter.error("#{e.class.name}: #{e}")
```

Comment:
Judging from what I've seen in the code, `Formatter.error([<label>, ]<message>)` would be more practical, as I don't think it makes sense to omit “label” (or more precisely the thing that should be styled). This is how it would be used and what it would yield:

```ruby
Formatter.error("Custom label", "Message") # => "_Custom label:_ Message"
Formatter.error("Error message") # => "_Error:_ Error message"
```

Here, `_<whatever>_` denotes the styled part. Would that work?